lib/features/composer/domain/model/email_request.dart

```diff
@@ -43,4 +44,49 @@ class EmailRequest with EquatableMixin {
     (emailActionType == EmailActionType.reply || emailActionType == EmailActionType.replyAll);
 
   bool get isEmailForwarded => emailIdAnsweredOrForwarded != null && emailActionType == EmailActionType.forward;
+
+  EmailRequest withUpdatedEmailHeaderMdn(
```

Comment:
Should update `headerMdn` in `EmailExtension` then callback at here because we can reuse it in the future

Reply:
done